python/lsst/ctrl/mpexec/cmdLineParser.py

```diff
@@ -91,6 +91,7 @@ def __call__(self, value):
 _ACTION_LABEL_TASK = _PipelineActionType("relabel", "(?P<label>.+):(?P<value>.+)")
 _ACTION_CONFIG = _PipelineActionType("config", "(?P<label>[^.]+)[.](?P<value>.+=.+)")
 _ACTION_CONFIG_FILE = _PipelineActionType("configfile", "(?P<label>.+):(?P<value>.+)")
+_ACTION_NAME_TEMPLATES = _PipelineActionType("name_templates", "(?P<label>[^.]+):(?P<value>.+)")
```

Comment:
That may not work too well if `value` contains `:` (as it should for dictionary). regexp is greedy by default and will match `:` with rightmost `:` in the string. Having unit test would probably catch it.